lib/fennec/udp/worker.ex

```diff
@@ -92,10 +91,27 @@ defmodule Fennec.UDP.Worker do
     handle_timeout(state)
   end
 
-  def handle_peer_data(:allowed, _ip, _port, _data, state) do
+  defp data_params(ip, port, data) do
+    alias Jerboa.Params, as: P
+    alias Jerboa.Format.Body.Attribute.{Data, XORPeerAddress}
+    import Fennec.Evaluator.Helper, only: [family: 1]
+    P.new()
+    |> P.put_class(:indication)
+    |> P.put_method(:data)
+    |> P.put_attr(%Data{content: data})
+    |> P.put_attr(%XORPeerAddress{
```

Comment:
As far as I know there is `new` constructor for all `XORAddress` structs that fills `family` for you. It's seems like a better way to go. It has been added to `Jerboa` very recently, so the `fennec` code didn't have a chance to adapt yet.